server/config.yaml

```diff
@@ -1,5 +1,10 @@
 clientId: $CLIENT_ID
 clientSecret: $CLIENT_SECRET
+projectId: $PROJECT_ID
```

Comment:
How do you plan to pass in the clientId, clientSecret and projectId?

Reply:
After setting the environment variable either locally or on App Engine, we can pass in the values when the `ServerChannel` is [initialized](https://github.com/googleinterns/dv360-excel-plugin/blob/da668522051694ff1a1579f647130f957cd7d4d1/server/lib/channel.dart#L25).